### PR TITLE
Add override option for dovecot

### PR DESCRIPTION
**Pull request by zurajm**
Added an option to configure overrides for dovecot
```yaml
dovecot:
  # enable dovecot overrides
  # overrides:
  #   dovecot.conf: |
  #     # More info here: https://mailu.io/1.8/kubernetes/mailu/index.html#dovecot
  #     mail_nfs_index = yes
  #     mail_nfs_storage = yes
  #     mail_fsync = always
  #     mmap_disable = yes
  #     mail_max_userip_connections=100
```
Might contribute to #22

### CODE REVIEW

**Comment by zurajm** [comment]
I was able to clean all loadBalancer stuff, I'm not sure what happened.

File: mailu/templates/dovecot.yaml [hunk 1]
```diff
@@ -145,4 +154,17 @@ spec:
     port: 4190
     protocol: TCP
 
+---
+
+{{- if .Values.dovecot.overrides }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: override-dovecot
```
Comment (micw) [inline mailu/templates/dovecot.yaml:163]: Please use the template for the name like in:
https://github.com/Mailu/helm-charts/blob/90d2e47ea62e5be4207b4aaa131fa668d0a2ee10/mailu/templates/dovecot.yaml#L8

Comment (micw) [reply mailu/templates/dovecot.yaml:163]: Also, please update the chart version to 0.1.1. The rest looks very good, I'd merge it then.

**Comment by zurajm** [comment]
Hi, fixed the name and bumped version.

**Comment by berni2288** [comment]
Just curious, what prevents this PR from being merged?

**Pull request merged**